src/databases/Blueprint/avtBlueprintDataAdaptor.C

```diff
@@ -677,22 +677,95 @@ HomogeneousShapeTopologyToVTKCellArray(const Node &n_topo,
     return ca;
 }
 
+// ****************************************************************************
+//  Method: UnstructuredTopologyToVTKUnstructuredGrid
+//
+//  Purpose:
+//   Constructs a vtkUnstructuredGrid from a Blueprint topology and coordset
```

Comment:
If you need to modify this again, you can add a period to this sentence and capitalize the first words in the sentences in the modification comments below.